__tests__/app.test.js

```diff
@@ -262,3 +262,24 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("status:200, should respond with an array of user objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { users } }) => {
+        expect(users).toBeInstanceOf(Array);
```

Comment:
You are using .forEach() array method, so checking if it's an array is not really necessary as it will fail anyway if it wasn't.

Reply:
Oh cool, gotcha. Makes sense! Thank you